apps/judicial-system/api/src/app/modules/case/interceptors/case.transformer.ts

```diff
@@ -204,10 +205,19 @@ export const getIndictmentDefendantsInfo = (theCase: Case) => {
       ? Date.now() >= new Date(verdictAppealDeadline).getTime()
       : false
 
+    const sentToPrisonAdminDate = theCase.defendantEventLogs
+      ?.filter(
+        (log) => log.caseId === theCase.id && log.defendantId === defendant.id,
+      )
+      .find(
+        (log) => log.eventType === DefendantEventType.SENT_TO_PRISON_ADMIN,
+      )?.created
+
```

Comment:
Please note that since the defendant event log is not used directly in the client it would make sense to push this logic all the way to an interceptor on the backend.